<launch>
 


    <node pkg="serial_manager" type="SerialManager" name="SerialManager" output="screen" >
	<remap from="/napodrone_pose" to="/aruco/pose_body_from_board"/>
        
	<param name="dev" type="string" value="/dev/ttyS1" />
	<param name="stream_pose" type="bool" value="True" />
	<param name="pose_el_time" type="int" value="500" />
    </node>

        
</launch>
